diff --git a/rust/src/schema.rs b/rust/src/schema.rs
index 29e8383..90ee90b 100644
--- a/rust/src/schema.rs
+++ b/rust/src/schema.rs
@@ -180,13 +180,13 @@ diesel::table! {
     }
 }
 
 diesel::table! {
     institution (id) {
         id -> Uuid,
-        links -> Array<Nullable<Jsonb>>,
+        links -> Array<Jsonb>,
         name -> Text,
         ms_tenant_id -> Nullable<Uuid>,
     }
 }
 
 diesel::table! {
@@ -269,13 +269,13 @@ diesel::table! {
     person (id) {
         id -> Uuid,
         first_name -> Text,
         last_name -> Text,
         email -> Text,
         institution_id -> Uuid,
-        roles -> Array<Nullable<UserRole>>,
+        roles -> Array<UserRole>,
         orcid -> Nullable<Text>,
         ms_user_id -> Nullable<Uuid>,
         api_key -> Nullable<Uuid>,
     }
 }
 
